components/gfx/Cargo.toml

```diff
@@ -20,7 +20,7 @@ euclid = { workspace = true }
 fnv = { workspace = true }
 fontsan = { git = "https://github.com/servo/fontsan" }
 gfx_traits = { workspace = true }
-harfbuzz-sys = "0.5"
+harfbuzz-sys = { version = "0.5", features = ["bundled"], git = "https://github.com/servo/rust-harfbuzz.git", rev = "2ce0a6d3e0cbdf36dedfbe1de7267e134c8d7446" }
```

Comment:
We can release a new version of harfbuzz to avoid having to use a git hash, but this is making a bigger change which is switching Servo to always using the "bundled" feature.

Reply:
right, `python\servo\command_base.py` only sets HARFBUZZ_SYS_NO_PKG_CONFIG (the predecessor of the bundled feature) for msvc and android.

Removed said defunct env var from command_base.py and enabled "bundled" only on said platforms.